Indico/Entity/Filter.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Collections.Generic;
+
+namespace Indico.Entity
+{
+    public abstract class Filter
+    {
+        protected List<dynamic> MergeFilters(IReadOnlyList<Filter> filters)
```

Comment:
Was it not possible to use a generic type parameter here instead of dynamic?

Reply:
anonymous types are dynamic, there is no other datatype to hold anonymous types